lib/multicell_location/services/skyhook_integration.c

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <modem/lte_lc.h>
+#include <modem/modem_info.h>
```

Comment:
In this case `cJSON` is not a hard dependency but rather added if enabled indirectly with `modem_info`.

Reply:
Added cJSON and CJSON_os explicitly